pylint_odoo/misc.py

```diff
@@ -271,7 +271,8 @@ def _skip_files_ext(self, fext, fnames):
         if fext != '.xml':
             return fnames
         info_called = [item[3] for item in inspect.stack() if
-                       'modules_odoo' in item[1]]
+                       'modules_odoo' in item[1]
+                       and item[3].startswith('_check_')]
```

Comment:
@moylop260 
Now I find in the stack call only the method start with `_check_` the method named `_check_` are the method related with one check

Reply:
and the tests that you had in the other commit?
In fact, you shouldn't squash commit of a PR in revision :(